packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -192,10 +189,28 @@ export interface ContainerRuntimeMessage {
     type: ContainerMessageType;
 }
 
+export interface ISummaryConfigurationCore {
```

Comment:
I used "Core" as an example. We need to find a better name for it

Reply:
Yes, I know - I tried to find a better name and the wait timedout so I kept it for now - will try to find again :)